tools/mkimage-dynamic-vhd/make-dynamic-vhd

```diff
@@ -17,7 +17,7 @@ cd files
 [ -t 0 ] || tar xf -
 
 INITRD="$(find . -name '*.img')"
-KERNEL="$(find . -name kernel -or -name '*bzImage')"
+KERNEL="./kernel"
 CMDLINE="$*"
 
 [ "$KERNEL" = "./kernel" ] || mv "$KERNEL" vmlinuz64
```

Comment:
This conditional should not be necessary anymore

Reply:
Yeah, I saw that, but didn't want to go through the "change it all, `lkt pkg push` it all, restart the 60-90 min CI again" cycle. :-) 

Will do it.